doc/source/code/Utilities.rst

```diff
@@ -8,10 +8,55 @@ Apart from the :doc:`operator <Operators>` classes, `eko` also provides some uti
   -  Implementation of the flavor number scheme and the quark thresholds both for
      the :class:`eko.strong_coupling.StrongCoupling` and the :doc:`operators <../theory/Matching>`
 
+  .. include:: IO-tabs/ThresholdConfig.rst
 
-.. include:: IO-tabs/ThresholdConfig.rst
+  When running in |VFNS| it is important to specify the number of flavors active at each given scale, since the evolution path
+  can be different depending of the chosen setting. This path is determined by :meth:`eko.thresholds.ThresholdsAtlas.path`.
 
+  Let us consider an example to better illustrate how it works.
```

Comment:
while an example is fine (and maybe even needed), let's not start with the most complicated and exotic one, but with a regular